### PR TITLE
use select field for ratings if IE11

### PR DESCRIPTION
**Pull request by DavidCramer**
Corrects the star rating on IE11 as mentioned in #8

### CODE REVIEW

**Comment by DavidCramer** [comment]
@kienstra You can give this a test. Thanks.

File: includes/functions.php [hunk 1]
```diff
@@ -158,22 +158,53 @@ function amp_travel_enqueue_styles() {
...
+	</fieldset>
 	</p>
 	<?php
 }
+
+/**
+ * Add select rating field to comments.
+ */
+function amp_travel_comment_select_rating_field() {
+	echo '<p class="comment-form-rating-select"><label for="rating">' . esc_html( 'Your rating' ) . '</label>
```
Comment (kienstra) [inline includes/functions.php]: Could this be:
```php
esc_html__( 'Your rating', 'travel' )
```

**Review by kienstra — CHANGES_REQUESTED** [review]
**Works Well, Very Minor Point**

Hi David,
Thanks for this good solution to IE 11 support. And thanks for reviewing #80 right away, after your workday.

There's a very minor point here.

Ratings on the single 'adventure' page now work on IE 11:
ie-11-rating

**Comment by DavidCramer** [comment]
@kienstra Thanks, changes applied and conflicts resolved.

**Review by kienstra — APPROVED** [review]

**Comment by kienstra** [comment]
Thanks, @DavidCramer! This looks good.

**Pull request merged**